docs/basic-operations.md

```diff
@@ -0,0 +1,105 @@
+<!--[metadata]>
++++
+title = "Basic Operations"
+description = "Basic operations using Notary"
+keywords = ["docker, notary, trust, image, signing, repository, cli"]
+[menu.main]
+parent="mn_notary"
++++
+<![end-metadata]-->
+
+# Basic operations using Notary
+
+The notary CLI provides a convenient way of managing trusted collections and the keys associated with them. On this page we describe a high level overview of the functionality of each subcommand. For a quick walk-through of the basic Notary functionality check out [getting started with Notary](gettingstarted.md).
```

Comment:
Yeah, that was my initial thought. Maybe that is the right way to consider it. I was thinking that each page should be considered a complete document, with links being the digital equivalent of references. Linking to a subsection within a page would then use quotes.